backend/src/Service/Admin/Report/ReportService.php

```diff
@@ -92,4 +96,23 @@ public function getDashboardStatisticsForAdmin(): array
 
         return $response;
     }
+
+    public function getCaptainsRatingsForAdmin(): array
+    {
+        $captainsRatings = $this->adminCaptainService->getCaptainsRatingsForAdmin();
+
+        if (count($captainsRatings) > 0) {
+            $response = [];
+
+            foreach ($captainsRatings as $key => $value) {
+                $response[$key] = $this->autoMapping->map('array', CaptainsRatingsForAdminGetResponse::class, $value);
+
+                $response[$key]->image = $this->uploadFileHelperService->getImageParams($value['imagePath']);
+            }
+
+            return $response;
+        }
+
+        return $captainsRatings;
```

Comment:
this is not clear, it should be something like an empty array or a specific response. it is okay for now but just for your record.